docs/taxonomy-revamp-2025.md

```diff
@@ -0,0 +1,93 @@
+[^3]: [yaml/pyyaml#486](https://github.com/yaml/pyyaml/issues/486)
+[^4]: You can experience this issue in action with the interactive experience on [yaml-multiline.info](https://yaml-multiline.info/).
+[^5]: [YAML Spec v1.2.2 on block scalar styles](https://yaml.org/spec/1.2.2/#81-block-scalar-styles)
+    > YAML provides two block scalar styles, literal and folded. Each provides a different trade-off between readability and expressive power.
+[^6]: [YAML Spec v1.2.2 on block chomping indicators](https://yaml.org/spec/1.2.2/#8112-block-chomping-indicator)
+    > Chomping controls how final line breaks and trailing empty lines are interpreted. YAML provides three chomping methods:
+[^7]: [YAML Spec v1.2.2 on block indentation indicators](https://yaml.org/spec/1.2.2/#8111-block-indentation-indicator)
+    > Every block scalar has a content indentation level. The content of the block scalar excludes a number of leading spaces on each line up to the content indentation level.
+    >
+    > If a block scalar has an indentation indicator, then the content indentation level of the block scalar is equal to the indentation level of the block scalar plus the integer value of the indentation indicator character.
+    >
+    > If no indentation indicator is given, then the content indentation level is equal to the number of leading spaces on the first non-empty line of the contents. If there is no non-empty line then the content indentation level is equal to the number of spaces on the longest line.
+    >
+    >It is an error if any non-empty line does not begin with a number of spaces greater than or equal to the content indentation level.
+    >
+    >It is an error for any of the leading empty lines to contain more spaces than the first non-empty line.
+    >
+    >A YAML processor should only emit an explicit indentation indicator for cases where detection will fail.
+[^8]: [YAML Spec v1.2.2 on the double-quoted flow scalar](https://yaml.org/spec/1.2.2/#double-quoted-style)
+    > In a multi-line double-quoted scalar, line breaks are subject to flow line folding, which discards any trailing white space characters. It is also possible to escape the line break character. In this case, the escaped line break is excluded from the content and any trailing white space characters that precede the escaped line break are preserved. Combined with the ability to escape white space characters, this allows double-quoted lines to be broken at arbitrary positions.
+[^9]: [YAML Spec v1.2.2 on the single-quoted flow scalar](https://yaml.org/spec/1.2.2/#single-quoted-style)
+    > In addition, it is only possible to break a long single-quoted line where a space character is surrounded by non-spaces. [...] All leading and trailing white space characters are excluded from the content. Each continuation line must therefore contain at least one non-space character. Empty lines, if any, are consumed as part of the line folding.
+[^10]: stuff
```

Comment:
Can you add a hyperlink to the stuff?

Reply:
Oh whoops. I forgot to finish filling that out 🤦‍♀️  Thank you for pointing that out.

Reply:
Fixed!